plumber-android/src/main/java/leakcanary/AndroidLeakFixes.kt

```diff
@@ -23,6 +23,7 @@ import java.util.EnumSet
 import java.util.concurrent.Executors
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.TimeUnit.SECONDS
+import java.util.concurrent.atomic.AtomicBoolean
```

Comment:
Is it unused? The flag seems to be a regular boolean

Reply:
oops prior work, fix imports needed. Should have looked at the diff before pushing.